test/thermodynamics/average_square_of_velocity_test.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.core.symbols.celsius import Celsius, to_kelvin_quantity
+from symplyphysics.laws.thermodynamics import average_square_of_velocity as velocity_law
+
+# Description
+## The ideal gas has a temperature of 298 K. The mass of an atom of a given gas is equal to 4 atomic units of mass. Then the average square of velocity should be 1.858 * 10^6 (m/s)^2
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    m = Quantity(4 * units.atomic_mass_unit)
+    temperature = Celsius(25)
```

Comment:
Description has temperature in Kelvin. Use numbers according to description to avoid additional check of input values.